openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx

```diff
@@ -1191,7 +1191,9 @@ const SVGIcons: FunctionComponent<Props> = ({ icon, ...props }: Props) => {
       // eslint-disable-next-line react/prop-types
       className={`svg-icon ${props.className ? props.className : ''}`}
       data-testid="image"
+      height="16px"
```

Comment:
This will make all icons of size 16px, also the monogram.
Can we double check this changes?
![image](https://user-images.githubusercontent.com/59080942/209095699-045fbbdf-f94a-4ede-b9dd-77768b22560f.png)

Reply:
The change was intentional. I have updated the size for the monogram now.  Can you please review now?